tools/create-translation-issues.sh

```diff
@@ -94,10 +96,10 @@ We are getting ready for the ${RELEASE} release and have created the [${TRANSLAT
 
 We would be happy if you updated the translations for **${LANG}** by **${DEADLINE}**.
 
-Please
+Please either [update the translations on Weblate](https://hosted.weblate.org/projects/jamulus/) or use Git:
 
 - Start your work in your fork on a branch based on jamuluswebsite'"'"'s `${TRANSLATE_BRANCH}` branch.
-- Update the language-specific files using your favorite editor (or directly on Github),
+- Update the language-specific files using your favourite editor (or directly on GitHub),
 - New/changed images are listed at the end of this issue. [Generate new URLs](https://github.com/jamulussoftware/jamuluswebsite/tree/release#adding-screenshots) for your image `.inc` files.
 - Commit and push your changes to your fork,
 - Open a Pull Request with your translations to the **${TRANSLATE_BRANCH}** branch with the subject `${TITLE}`,
```

Comment:
```suggestion
- Open a pull request with your translations to the **${TRANSLATE_BRANCH}** branch with the subject `${TITLE}`,
```

Reply:
Is that the correct spelling? I think this should be another commit.

Reply:
It is the convention GitHub uses https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests
Seems correct to me.